src/NosCore.GameObject/Character.cs

```diff
@@ -430,8 +435,8 @@ public void ChangeClass(CharacterClassType classType)
 
             JobLevel = 1;
             JobLevelXp = 0;
-            SendPacket(new NpInfoPacket());
-            SendPacket(new PclearPacket());
+            await SendPacket(new NpInfoPacket());
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add '.ConfigureAwait(false)' to this call to allow execution to continue in any thread.](https://app.codacy.com/manual/NosCoreIO/NosCore/pullRequest?prid=5262615)